bigbluebutton-html5/public/locales/en.json

```diff
@@ -1405,6 +1405,9 @@
     "app.learningDashboard.pollsTable.anonymousRowName": "Anonymous",
     "app.learningDashboard.pollsTable.noPollsCreatedHeading": "No polls have been created",
     "app.learningDashboard.pollsTable.noPollsCreatedMessage": "Once a poll has been sent to users, their results will appear in this list.",
+    "app.learningDashboard.pluginsTable.title": "Plugins",
+    "app.learningDashboard.pluginsTable.noPluginsCreatedHeading": "No plugin yet for this meeting",
+    "app.learningDashboard.pluginsTable.noPluginsCreatedMessage": "Once a plugin stores its data into learning-analytics-dashboard, it will appear in this list.",
```

Comment:
I would say to simply doesn't show the Plugins panel if it has no data for plugins.